pkg/controllers/jobflow/jobflow_controller_action.go

```diff
@@ -63,7 +65,26 @@ func (jf *jobflowcontroller) syncJobFlow(jobFlow *v1alpha1flow.JobFlow, updateSt
 	}
 	jobFlow.Status = *jobFlowStatus
 	updateStateFn(&jobFlow.Status, len(jobFlow.Spec.Flows))
-	_, err = jf.vcClient.FlowV1alpha1().JobFlows(jobFlow.Namespace).UpdateStatus(context.Background(), jobFlow, metav1.UpdateOptions{})
+
+	err = retry.RetryOnConflict(retry.DefaultRetry, func() error {
```

Comment:
Added the retry mechanism after resource version conflicts to avoid the next reconcile.

Reply:
It's truly a problem, but this way seems will be more time consuming.